src/app/page/Home/VolunteerHome.jsx

```diff
@@ -47,68 +76,78 @@ export default function VolunteerHome({ currentUser }) {
+    updateInProgressMissions(inProgressMissions.concat(mission));
+    updateInPlanningMissions(inPlanningMissions.filter((m) => m.id !== missionId));
   };
 
   const handleStartMissionFromSuggested = (missionId) => {
     User.startMission(currentUser.uid, missionId);
-    const mission = suggestedMissions.filter((m) => m.id === missionId);
-    updateStartedMissions(startedMisssions.concat(mission));
-    updateSuggestedMissions(suggestedMissions.filter((m) => m.id !== missionId));
+    const mission = proposedMissions.filter((m) => m.id === missionId);
+    updateInProgressMissions(inProgressMissions.concat(mission));
+    updateProposedMissions(proposedMissions.filter((m) => m.id !== missionId));
   };
 
   const handleDeliveringMissionsFromStarted = (missionId) => {
     User.deliverMission(currentUser.uid, missionId);
-    const mission = startedMisssions.filter((m) => m.id === missionId);
+    const mission = inProgressMissions.filter((m) => m.id === missionId);
 
-    updateCompletedMissions(completedMisssions.concat(mission));
+    updateCompletedMissions(completedMissions.concat(mission));
 
-    updateStartedMissions(startedMisssions.filter((m) => m.id !== missionId));
+    updateInProgressMissions(inProgressMissions.filter((m) => m.id !== missionId));
   };
+
+  const proposedLabel = "Proposed (" + proposedMissions.length + ")";
+  const planningLabel = "Planning (" + inPlanningMissions.length + ")";
+  const inProgressLabel = "In Progress (" + inProgressMissions.length + ")";
+  const completedLabel = "Completed (" + completedMissions.length + ")";
+
   return (
     <div className={classes.root}>
-      <AppBar position="static">
-        <Tabs
-          value={value}
-          variant="fullWidth"
-          centered
-          onChange={handleChange}
-          aria-label="volunteer dashboard tabs"
-        >
-          <Tab label="Started" {...a11yProps(0)} />
-          <Tab label="Assigned" {...a11yProps(1)} />
-          <Tab label="Suggested" {...a11yProps(2)} />
-          <Tab label="Completed" {...a11yProps(3)} />
-        </Tabs>
-      </AppBar>
+      <ThemeProvider theme={theme}>
+        <AppBar position="static">
+          <Tabs
+            TabIndicatorProps={{ style: { background: "#373985" } }}
+            classes={classes}
+            value={value}
+            variant="fullWidth"
+            centered
+            onChange={handleChange}
+            aria-label="volunteer dashboard tabs"
+          >
+            <Tab label={proposedLabel} {...a11yProps(2)} />
+            <Tab label={planningLabel} {...a11yProps(1)} />
+            <Tab label={inProgressLabel} {...a11yProps(0)} />
+            <Tab label={completedLabel} {...a11yProps(3)} />
+          </Tabs>
+        </AppBar>
+      </ThemeProvider>
       <TabPanel value={value} index={0}>
         <VolunteerHomeMissionList
-          missions={startedMisssions}
+          missions={inProgressMissions}
```

Comment:
Can we make sure the TabPanel component mirrors the order of the Tab component? If ProposedTab is at index 0, TabPanel for Proposed Mission should be at 0.